addons/stock/models/stock_orderpoint.py

```diff
@@ -422,7 +423,21 @@ def _get_orderpoint_values(self, product, location):
             'trigger': 'manual',
         }
 
-    def _get_replenishment_order_notification(self):
+    def _get_replenishment_order_notification(self, written_after):
```

Comment:
@odony the signature of this method have been changed in stable version, can you check ?

Reply:
@fmdl Sorry, I will revert it and use the context instead